Testing/Heroes/GargoyleTests.cs

```diff
@@ -0,0 +1,241 @@
+        [Test()]
+        public void TestGargoyleInnatePower()
+        {
+            StartTestGame();
+
+            GoToUsePowerPhase(gargoyle);
+
+            // Select a target. 
+            DecisionSelectTarget = baron.CharacterCard;
+            UsePower(gargoyle.CharacterCard);
+                        
+            // Reduce the next damage it deals by 1.
+            QuickHPStorage(gargoyle.CharacterCard);
+            DealDamage(baron.CharacterCard, gargoyle.CharacterCard, 1, DamageType.Melee);
+            QuickHPCheckZero();
+
+            // Increase the next damage Gargoyle deals by 1.
+            QuickHPStorage(baron.CharacterCard);
+            DealDamage(gargoyle.CharacterCard, baron.CharacterCard, 1, DamageType.Melee);
+            QuickHPCheck(-2);
+        }
+
+        /* 
+         * Incap 1
+         * "Increase the next damage dealt by a hero target by 2."
+        */
+        [Test()]
+        public void TestGargoyleIncap1HeroCharacter()
+        {
+            SetupIncapTest();
+            UseIncapacitatedAbility(gargoyle, 0);
+
+            // Make sure it only affects hero damage
+            QuickHPStorage(baron);
+            DealDamage(bunker.CharacterCard, baron, 1, DamageType.Melee);
+            QuickHPCheck(-3);
+
+            // Should only be the next damage,  this packet of damage should not be increased
+            QuickHPStorage(baron);
+            DealDamage(bunker.CharacterCard, baron, 1, DamageType.Melee);
+            QuickHPCheck(-1);
+        }
+        [Test()]
+        public void TestGargoyleIncap1HeroTarget()
+        {
+            Card mrChomps;
+
+            SetupIncapTest();
+            UseIncapacitatedAbility(gargoyle, 0);
+
+            mrChomps = PlayCard(unity, "RaptorBot");
+
+            // Make sure it only affects hero damage
+            QuickHPStorage(baron);
+            DealDamage(mrChomps, baron, 1, DamageType.Melee);
+            QuickHPCheck(-3);
+
+            // Should only be the next damage,  this packet of damage should not be increased
+            QuickHPStorage(baron);
+            DealDamage(mrChomps, baron, 1, DamageType.Melee);
+            QuickHPCheck(-1);
+        }
+        [Test()]
+        public void TestGargoyleIncap1Villain()
+        {
+            SetupIncapTest();
+            UseIncapacitatedAbility(gargoyle, 0);
+
+            // Make sure it isn't affecting the villians damage
+            QuickHPStorage(bunker);
+            DealDamage(baron.CharacterCard, bunker, 1, DamageType.Melee);
+            QuickHPCheck(-1);
+        }
+        [Test()]
+        public void TestGargoyleIncap1Environment()
+        {
+            TurnTakerController megalopolis;
+            Card plummetingMonorail;
+
+            SetupIncapTest();
+            UseIncapacitatedAbility(gargoyle, 0);
+
+            megalopolis = FindEnvironment();
+            Assert.IsNotNull(megalopolis);
+
+            plummetingMonorail = PlayCard(megalopolis, "PlummetingMonorail");
```

Comment:
PlayCard will assert on it's own if the Identifier is wrong, no need for a separate Assert